sass/cd/cd-header/_cd-nav.scss

```diff
@@ -75,6 +70,8 @@
   float: left;
 
   .cd-nav {
+    @include z-index(main-menu); // zindex may need adjusting depending on other site content.
```

Comment:
@rupl FYI You might notice the main menu dropdowns do not appear because of a z-index issue
 https://commondesign.demo.ahconu.org
Until this PR is merged, you can either remove the cd-hero include in page--front.html.twig
https://github.com/UN-OCHA/common_design/pull/25/files#diff-841e2d37dcc55695d28dffc8974728c6R58
or add `z-index: 199` for .cd-nav